README.md

```diff
@@ -1,62 +1,72 @@
 # html5-to-pdf-with-Puppeteer
 
-Convert HTML5 page to pdf or image with Puppeteer
+## Description
 
-## Start the server for production
+this project is about to Convert HTML5 page to pdf or image with Puppeteer
```

Comment:
@steve3524  A remplacer par "This project is about Converting HTML5 page to pdf or image with Puppeteer"

Reply:
@Didis29  Corrigé

Reply:
@steve3524 Pourtant je vois écrit "this project is about to Convert HTML5 page to pdf or image with Puppeteer" au lieu de "This project is about Converting HTML5 page to pdf or image with Puppeteer" ?

Reply:
@steve3524 Commentaire oublié

Reply:
En réalité, les deux phrases ont exactement la même signification c est pour ça j'ai pas trouver nécessaire de changer

Reply:
@steve3524 Pas vraiment mais bon ^^'